components/appmgt/org.wso2.carbon.appmgt.usage/org.wso2.carbon.appmgt.usage.publisher/src/main/java/org/wso2/carbon/appmgt/usage/publisher/APPMgtUiActivitiesBamDataPublisher.java

```diff
@@ -232,20 +220,18 @@ public void publishUserActivityEvents(String action, String item,
 				}
 			} else {
 				// Write directly to DB
-				AppMDAO.saveStoreHits(appId.trim(), userId.trim(), tenantId,
-						strDataContext);
-			}
+                AppMDAO.saveStoreHits(appId.trim(), userId.trim(), tenantId);
+            }
 		} catch (AgentException e) {
-			// Here I'm only logging the exceptions but not throwing externally
-			// as this method is used only to store the store hit count, An
-			// exception in this method should not effect/block the users other
-			// actions
-			log.error("Failed to publish build event : " + strDataContext
-					+ " : " + e.getMessage(), e);
-		} catch (AppManagementException e) {
-			log.error("Failed to write to table : " + e.getMessage(), e);
-		} catch (SQLException e) {
-			log.error("SQL exception found : " + e.getMessage(), e);
-		}
-	}
+             // Here I'm only logging the exceptions but not throwing externally
+             // as this method is used only to store the store hit count, An
+             // exception in this method should not effect/block the users other
+             // actions
+             log.error("Failed to publish build event : " + e.getMessage(), e);
+        } catch (AppManagementException e) {
+            log.error("Failed to write to table : " + e.getMessage(), e);
```

Comment:
Here the exception is only logged (but not thrown externally) as this method is used only just to store the store hit count, An exception in this method should not effect/block the users other actions (UI activities).